invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -426,27 +541,106 @@ class CommunityProfileForm extends Component {
                       []
                     )}
                     serializeSuggestions={(organizations) =>
-                      _map(organizations, (organization) =>
-                      {
-                        const isKnownOrg = this.knownOrganizations.hasOwnProperty(organization.name);
+                      _map(organizations, (organization) => {
+                        const isKnownOrg =
+                          this.knownOrganizations.hasOwnProperty(
+                            organization.name
+                          );
                         if (!isKnownOrg) {
                           this.knownOrganizations = {
                             ...this.knownOrganizations,
-                            [organization.name]: organization.id
+                            [organization.name]: organization.id,
                           };
                         }
                         return {
                           text: organization.name,
                           value: organization.name,
-                          key: organization.name
-                        }
+                          key: organization.name,
+                        };
                       })
                     }
                     label={i18next.t("Organizations")}
                     noQueryMessage={i18next.t("Search for organizations...")}
                     allowAdditions
                     search={(filteredOptions, searchQuery) => filteredOptions}
                   />
+                  <FundingField
+                    fieldPath="metadata.funding"
+                    searchConfig={{
+                      searchApi: {
+                        axios: {
+                          headers: {
+                            //  FIXME use for internationalisation
```

Comment:
Shall we fix that? Or create an issue and delete the comment?

Reply:
I think the backend for funders and grants is not ready yet to internationalise (e.g. `funder.title` should be declared as a `L10NString`).